### ISSUE TITLE
mAP scores far lower on a specific class than the overall MS COCO 2017 

### ISSUE THREAD

**Issue by AlexTS1980** [matterport/Mask_RCNN#397]
I've run an out-of-the-box Mask-RCNN (mask_rcnn_coco.h5 weights) on the Cow class from MS COCO 2017 validation set (instance segmentation), it's a total of 87 images (only masks of Cow instances in the ground truth). I'm getting a suspiciously low precision: 0.36@50% IoU and 0.29 mAP, which is far lower than the other results I've found (e.g. around 0.6 and 0.37 accordingly). Why is this the case? To get AP, I used the `compute_ap` function from MRCNN utils. The config file is use is the same excecpt the following hyperparameters:

```
   GPU_COUNT = 0
   DETECTION_MIN_CONFIDENCE = 0.5
   IMAGES_PER_GPU = 1
   NUM_CLASSES = 80 + 1
   IMAGE_MIN_DIM = 640
   IMAGE_MAX_DIM = 640
   VALIDATION_STEPS = 1
```
I also set DETECTION_MIN_CONFIDENCE to 0.7, but the changes were miniscule.
